src/DynamicHDT.cpp

```
@@ -18,23 +18,23 @@ std::string hdt::util::UInt32toString(uint32_t formID)
 {
-	std::string str_no_prefix = str_with_prefix;
+	std::string str_no_prefix{ str_with_prefix };
 	if (str_with_prefix.find("hdtSSEPhysics_AutoRename_") == 0) {
 		str_no_prefix = str_with_prefix.substr(str_with_prefix.find(' ') + 1);
 	}
 
 	return str_no_prefix;
 }
 
-bool _match_name(hdt::IDStr& a, hdt::IDStr& b)
+bool _match_name(const RE::BSFixedString& a, const RE::BSFixedString& b)
```

Clang (14.0.6)

[warning] declaration uses identifier '_match_name', which is reserved in the global namespace

(bugprone-reserved-identifier)